doc/source/whatsnew/v0.18.2.txt

```diff
@@ -101,6 +101,7 @@ API changes
 ~~~~~~~~~~~
 
 
+- ``DataFrame.to_csv()`` has dropped the ``engine`` parameter (:issue:`13418`)
```

Comment:
it's a prior deprecation removal / so no

that would totally break if you had ignored the warning and not fixed your code

people should have done it - but can't hurt to put in major version